pkg/proxy/apis/config/types.go

```diff
@@ -179,10 +179,15 @@ type KubeProxyConfiguration struct {
 	// kube-proxy is running on. If unset, the node name is assumed to be the same as
 	// the node's hostname.
 	HostnameOverride string
-	// bindAddress can be used to override kube-proxy's idea of what its node's
-	// primary IP is. Note that the name is a historical artifact, and kube-proxy does
-	// not actually bind any sockets to this IP.
-	BindAddress string
+	// nodeIPOverride can be used to override kube-proxy's idea of what its node's
+	// primary IPs are. nodeIPOverride[0] will be used for defaulting healthzBindAddress
+	// and metricsBindAddress values. If nodePortAddresses is set to "primary" node-ports
+	// will be exposed on the given ips.
+	NodeIPOverride []string
+	// ipFamilyPolicy controls the nodeIP detection logic, which further influences if LoadBalancer
+	// services should be allowed on node when SourceRanges are set and on which interfaces node
+	// ports should be exposed when nodePortAddresses is set to 'primary'.
+	IPFamilyPolicy IPFamilyPolicy
```

Comment:
"ipFamilyPolicy controls whether kube-proxy runs in single-stack or dual-stack mode. The default is 'PreferDualStack' if nodeIPOverride is not specified, 'SingleStack' if nodeIPOverride contains a single IP, or 'RequireDualStack' if nodeIPOverride contains dual-stack IPs."

(But also, you don't seem to be defaulting it?)

Reply:
but then it never gets defaulted for v1alpha2?